script.js

```diff
@@ -0,0 +1,46 @@
+ var box = $(".box");
+ var activeBox = $(".box").eq("")
+var counter= 0;
+var winningBoxes=[
+[box.eq("0"),box.eq("1"),box.eq("2")],
```

Comment:
Right now this only has a combination of the top row, is that hardcoded as a test?